server/db/create_db.js

```diff
@@ -6,69 +6,80 @@
 
 // TODO extract schema into, you know, a schema file
 // TODO figure out migrations story - mysql-patcher? node-migrate?
+// TODO normalize URLs and store in visits 'url' field
+// TODO add to visits when addon supports it: referrer, searchTerms, engagementTime
 
-var mysql = require('mysql');
+var program = require('commander');
+var utils = require('./utils');
 var config = require('../config');
 var log = require('../logger')('server.db');
 
-// TODO use connection pooling
-var conn = mysql.createConnection({
-  host: config.get('db.mysql.host'),
-  user: config.get('db.mysql.user'),
-  password: config.get('db.mysql.password')
-});
+var pool = utils.createPool(true);
 
 function createDatabase(cb) {
   var dropDatabaseQuery = 'DROP DATABASE IF EXISTS chronicle';
   var createDatabaseQuery = 'CREATE DATABASE IF NOT EXISTS chronicle ' +
     'CHARACTER SET utf8 COLLATE utf8_unicode_ci';
   var useDatabaseQuery = 'USE chronicle';
-  var createTableQuery = 'CREATE TABLE IF NOT EXISTS users (' +
-    'fxa_id BINARY(16) NOT NULL PRIMARY KEY,' +
```

Comment:
Sounds good. We can do some testing when we have much datas to see if it would be useful to use binary instead.